oeps/processes/oep-0001.rst

```diff
@@ -133,14 +139,18 @@ OEP Workflow
 Submitting an OEP
 -----------------
 
-Step 1. Request an Arbiter
+Step 1. Find an Arbiter
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Request an Arbiter from the `edX Architecture Team`_ by emailing
-`arch-team@edx.org`_. This Arbiter will be recorded in the "Arbiter" header on the
-OEP.
+When writing an OEP, you may already have an idea of an Arbiter in mind. If so,
+reach out to that person and ask them; they should have the domain expertise
+needed to be an effective Arbiter and the time to do so. It is best practice for
+the Arbiter to be from a different team or group than the author.
 
-.. _`arch-team@edx.org`: mailto:arch-team@edx.org
+If you're not sure who would make a good Arbiter, you should post in the
+`Architecture Group`_ category on the Discourse forum or the ``#architecture``
+channel in the `Open edX Slack`_. Once found, this Arbiter will be
+recorded in the "Arbiter" header on the OEP.
```

Comment:
Should we include anything about what to do if you disagree with an arbiter choice?  Having some text to guide people if they have concerns about a specific arbiter, might be useful as a way of mediate issues.

An initial stab at what it might look like:
"If you have concerns about an abiter that has been chosen for a particular OEP, please share your concerns with the author first and see if you can resolve your concerns directly.  If you continue to have concerns, please share them in slack or discourse..."

Reply:
Thanks. I added this, and also a link to our code of conduct (where you can get in touch w/ specific people if you have major concerns)